.github/workflows/cypress_workflow.yml

```diff
@@ -128,14 +173,26 @@ jobs:
           done
           echo "SPEC=${FORMATTED_SPEC}" >> $GITHUB_ENV
 
+      # Setup spec files for Dashboards in-house cypress tests
+      - name: Setup spec files for Dashboards tests
+        if: ${{ inputs.specs == '' && matrix.test_location == 'source' }}
+        shell: bash
+        run: |
+          IFS="," read -a SPEC_ARRAY <<< $(yarn --silent osd:ciGroup${{ matrix.group }})
+          DASHBOARDS_SPEC=''
+          for i in "${SPEC_ARRAY[@]}"; do
+          DASHBOARDS_SPEC+="cypress/integration/core_opensearch_dashboards/${i},"
```

Comment:
we desperately need to be able to tag cypress test so that when we write them inhouse the test can decide which workflow it will run in